packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.spec.js

```diff
@@ -1,18 +1,70 @@
 import { shallow } from 'enzyme'
-import React from 'react'
+import { ProjectAboutPage, PageHeading } from './ProjectAboutPage'
+import AboutDropdownNav from './components/AboutDropdownNav'
+import AboutSidebar from './components/AboutSidebar'
+import TeamMember from './components/TeamMember'
 
-import ProjectAboutPage from './ProjectAboutPage'
+let wrapper
```

Comment:
We've learned this the hard way, but it's best to declare this within the describe scope rather than as a global. There can be unintended side effects otherwise with test running, if say someone forgets to declare wrapper and start using it and there's this global reference still hanging around.

There's a global leak detection we could enable for mocha since I think we still have a few of these around... and that's likely not helpful for you if you come across it as a reference!

Reply:
That totally makes sense! I'll be sure to move the declaration before merging.